doc/topics/cloud/profitbricks.rst

```diff
@@ -199,6 +203,9 @@ public_lan
     LAN exists, then a new public LAN will be created. The value accepts a LAN
     ID (integer).
 
+public_ips
+    Public IPs assigned to the NIC in the public LAN.
+
```

Comment:
Adding a `.. versionadded:: Fluorine` directive would be good for this new option.